### PR TITLE
lv_conf_kconfig: Update

### PR DESCRIPTION
**Pull request by C47D**
Update lv_conf_kconfig with some missing symbol definitions.

### CODE REVIEW

File: src/lv_conf_kconfig.h [hunk 1]
```diff
@@ -10,35 +10,24 @@
...
-  #define LV_THEME_DEFAULT_INIT               lv_theme_empty_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_empty_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_TEMPLATE)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_template_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_template_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MATERIAL)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_material_init
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_material_init
  #elif defined (CONFIG_LV_THEME_DEFAULT_INIT_MONO)
-  #define LV_THEME_DEFAULT_INIT               lv_theme_mono_init
```
Comment (kisvegabor) [inline src/lv_conf_kconfig.h:25]: This line also needs to be changed, isn't it?

**Review by kisvegabor — COMMENTED** [review]
Why is it better to define the `CONFIG_...` version here instead of the "normal" LVGL defines?

E.g. 
```c
  #define LV_THEME_DEFAULT_FONT_SMALL         &lv_font_montserrat_30

  #define CONFIG_LV_THEME_DEFAULT_FONT_SMALL         &lv_font_montserrat_30  //Why is it better?
```

**Comment by C47D** [comment]
I did it just to keep all the definitions at `lv_conf_internal`. I tried to keep `lv_conf_internal` unchanged from what's automatically generated.

**Comment by C47D** [comment]
I can revert to normal LV defines instead of CONFIG if you think it's better.

**Comment by kisvegabor** [comment]
Actually, both can work. The only difference is that it will be handled by the first or second line of the `#if...`s.
```c

#ifndef LV_THEME_DEFAULT_FONT_SMALL
#  ifdef CONFIG_LV_THEME_DEFAULT_FONT_SMALL
#    define LV_THEME_DEFAULT_FONT_SMALL CONFIG_LV_THEME_DEFAULT_FONT_SMALL
#  else
#    define  LV_THEME_DEFAULT_FONT_SMALL         &lv_font_montserrat_14
#  endif
#endif
```

I'm sure you are more immersed in the Kconfig topic so probably you can judge better which one could be better. :slightly_smiling_face: So both are equally good for me.

File: src/lv_conf_kconfig.h (hunk 1 above)
Comment (C47D) [reply src/lv_conf_kconfig.h:25]: It's changed below, see my other comment.

File: src/lv_conf_kconfig.h [hunk 2]
```diff
@@ -10,35 +10,36 @@
...
-#ifdef LV_THEME_DEFAULT_COLOR_PRIMARY
- #ifdef CONFIG_LV_THEME_DEFAULT_COLOR_PRIMARY
-  #define LV_THEME_DEFAULT_COLOR_PRIMARY lv_color_hex(CONFIG_LV_THEME_DEFAULT_COLOR_PRIMARY) 
- #endif
-#endif
- 
-#ifdef LV_THEME_DEFAULT_COLOR_SECONDARY
- #ifdef CONFIG_LV_THEME_DEFAULT_COLOR_SECONDARY
-  #define LV_THEME_DEFAULT_COLOR_SECONDARY lv_color_hex(CONFIG_LV_THEME_DEFAULT_COLOR_SECONDARY) 
+  #define CONFIG_LV_THEME_DEFAULT_INIT               lv_theme_mono_init
```
Comment (C47D) [inline src/lv_conf_kconfig.h:42]: Here's the definition of `CONFIG_LV_THEME_DEFAULT_INIT` when the mono theme is selected.

**Comment by kisvegabor** [comment]
All looks good, thanks.

Is it ready to merge?

**Pull request merged**